README.md

```diff
@@ -63,7 +63,7 @@ fi
 ## Building the Emulator
 
 Goggle source code can be built natively on the host machine and used for debugging.
-Requires build-essential tools and SDL2 development libraries to be already installed.
+Requires build-essential tools and libsdl2-dev development libraries to be already installed.
```

Comment:
While this maybe true for debian based distros this is not true for others.  I would stick with the original description.